### ISSUE TITLE
Kinect 2 Framerate

### ISSUE THREAD

**Issue by stefie10** [Kinovarobotics/kinova-movo#30]
When running on a remote machine, the Kinect 2 framerate is something like 1hz.  This is not even the point cloud, but the framerate I get when running rostopic hz on one of the image topics.  When I run the same command on the movo2 machine, I get 30hz, which is more like I expect.

We see an improvement when running on the wired network (something like 15 hz), but I expect the wireless framerate to be somewhat similar.  Has anyone else experienced this problem?  Are there suggested fixes?

**Comment by cbames** [comment]
What are the wireless devices on the movo? We're upgrading anathema to WIFI
AD, in hopes that we'll have enough bandwidth to get 15hz wirelessly.

Barrett Ames

**Comment by tttor** [comment]
This may be relevant: https://github.com/orgs/Kinovarobotics/teams/movo-beta-program/discussions/28

**Comment by martine1406** [comment]
Indeed, the discussion pointed out by @tttor describes well the current situation on the movo I think. It also explains why we believe the framerate is so slow.
